webapp/pages/moderation/index.vue

```diff
@@ -42,25 +63,15 @@
           </nuxt-link>
         </div>
       </template>
-      <template slot="type" slot-scope="scope">
-        <ds-text color="soft">
-          <ds-icon
-            v-if="scope.row.type === 'Post'"
-            v-tooltip="{ content: $t('report.contribution.type'), placement: 'right' }"
-            name="bookmark"
-          />
-          <ds-icon
-            v-else-if="scope.row.type === 'Comment'"
-            v-tooltip="{ content: $t('report.comment.type'), placement: 'right' }"
-            name="comments"
-          />
-          <ds-icon
-            v-else-if="scope.row.type === 'User'"
-            v-tooltip="{ content: $t('report.user.type'), placement: 'right' }"
-            name="user"
-          />
-        </ds-text>
+      <!-- reasonCategory -->
+      <template slot="reasonCategory" slot-scope="scope">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-14T19:33:29Z" title="Monday, October 14th 2019, 9:33:29 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -42,25 +63,15 @@
           </nuxt-link>
         </div>
       </template>
-      <template slot="type" slot-scope="scope">
-        <ds-text color="soft">
-          <ds-icon
-            v-if="scope.row.type === 'Post'"
-            v-tooltip="{ content: $t('report.contribution.type'), placement: 'right' }"
-            name="bookmark"
-          />
-          <ds-icon
-            v-else-if="scope.row.type === 'Comment'"
-            v-tooltip="{ content: $t('report.comment.type'), placement: 'right' }"
-            name="comments"
-          />
-          <ds-icon
-            v-else-if="scope.row.type === 'User'"
-            v-tooltip="{ content: $t('report.user.type'), placement: 'right' }"
-            name="user"
-          />
-        </ds-text>
+      <!-- reasonCategory -->
+      <template slot="reasonCategory" slot-scope="scope">
```


Looks like the 'slot-scope' attribute is deprecated in VueJS 2.6.0+ https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute